### PR TITLE
Update b2b nextjs example

### PR DESCRIPTION
**Pull request by mffap**
- Add YT video of the demo
- Provide step-by-step instructions
- Update links to zitadel.com and remove caos branding

### CODE REVIEW

File: b2b-nextjs/README.md [hunk 1]
```diff
@@ -99,3 +99,32 @@ Users with `view` role can view granted projects on their organization which wer
...
+In the `B2B-Demo` delegate access management of the project `Portal` to `B2B-Demo-Customer` as described above. You don't see any other granted projects because the user is not authorized, yet. Log out.
+
+### 3. Authorize a user for the granted project
+
+Grant your user the role `reader` to the granted project `Portal`. Login again. You should see `Portal` in the tab "Granted Projects". You are not allowed to select the tab "Authorization", this is only for admins. You don't need to logout for the next step.
+
+### 4. Grant and authorize another project
+
+In the `B2B-Demo` delegate access management of the project `Data Cube` to `B2B-Demo-Customer` as described above. As soon as you granted the project, authorize your user to that project by assigning some roles. The new project should load on the Portal.  
+This is how you can use a Service User for Client-Discovery.
```
Comment (fforootd) [inline b2b-nextjs/README.md:125]: Does this really relate to a service user? IMO that's just the users access rights

File: b2b-nextjs/README.md [hunk 2]
```diff
@@ -99,3 +99,32 @@ Users with `view` role can view granted projects on their organization which wer
...
+
+Watch the [Youtube Video](https://www.youtube.com/embed/-BVgq3mmxGE) for a walk-through.
+
+### 1. Validate Setup
+
+Login with your user. You should have no granted projects and no roles in this organization. Log out.
+
+### 2. Grant a project
+
+In the `B2B-Demo` delegate access management of the project `Portal` to `B2B-Demo-Customer` as described above. You don't see any other granted projects because the user is not authorized, yet. Log out.
```
Comment (fforootd) [inline b2b-nextjs/README.md:116]: Maybe add a quick hint why the logout is needed (because we use the token infos and not call the api)

Comment (mffap) [reply b2b-nextjs/README.md:116]: Hint added. Valid point bc this might not be feasible for prod use.

File: b2b-nextjs/README.md (hunk 1 above)
Comment (mffap) [reply b2b-nextjs/README.md]: the service user uses management/v1/projectgrants/_search to discover the services available to the organization. I've clarified it with a link to the source code.

Comment (fforootd) [reply b2b-nextjs/README.md]: But that call does happen with the currently logged In user not the service user, right?

Comment (mffap) [reply b2b-nextjs/README.md]: The request to the management api uses an auth token `Bearer ${token.access_token}`. That token comes from `requestAccessToken` in the `/lib/jwt.ts`. `requestAccessToken` uses the service user to request an auth token from zitadel. That's why I assumed that the service user is used for querying this info from the management api.

**Comment by peintnermax** [comment]
I've removed the userid header from /api/grantedprojects to not confuse readers. As mentioned above, the service user reads all projectgrants independenty, so no context of the user is needed.

File: b2b-nextjs/README.md (hunk 1 above)
Comment (mffap) [reply b2b-nextjs/README.md]: Ok I've remove the part about service discovery from the step-by-step guide. Added the keyword to the top.

**Comment by mffap** [comment]
@peintnermax I've removed the lines that were discussed with @fforootd . Can you please approve this PR, if the rest looks ok?

**Review by peintnermax — APPROVED** [review]

**Pull request merged**